src/routes/artist.router.ts

```diff
@@ -1,10 +1,11 @@
 import { Router } from 'express';
 
-import { createArtist } from '../controllers/artist.controller';
+import { createArtist, getArtists } from '../controllers/artist.controller';
 import { signupValidation, validate } from '../middlewares';
 
 const router = Router();
 
 router.post('/signup', signupValidation, validate, createArtist);
+router.get('/all', getArtists);
```

Comment:
According to the spec
```suggestion
router.get('/', getArtists);
```